Add WAF exclision blocks on Azure AppGateway module (#1284)
prefapp/tfm: modules/azure-app-gateway/README.md

}
```

## WAF exclusions example

> Use this shape inside `web_application_firewall_policy` when you need to preserve or define managed rule exclusions:

Reviewer: ```suggestion
Add this config to the `web_application_firewall_policy` block when you need to define managed rule exclusions:
```
